source/slang/slang-options.cpp

```diff
@@ -1912,6 +1934,8 @@ SlangResult OptionsParser::_parse(
                 linkage->m_optionSet.set(optionKind, compressionType);
                 break;
             }
+            case OptionKind::EmbedDXIL: SLANG_RETURN_ON_FAIL(addEmbeddedLibrary(CodeGenTarget::DXIL)); break;
+            case OptionKind::EmbedSPIRV: SLANG_RETURN_ON_FAIL(addEmbeddedLibrary(CodeGenTarget::SPIRV)); break;
```

Comment:
I think it will be easier to read if these are broken into multiple lines,
```
    case OptionKind::EmbedDXIL:
        SLANG_RETURN_ON_FAIL(addEmbeddedLibrary(CodeGenTarget::DXIL));
        break;
    case OptionKind::EmbedSPIRV:
        SLANG_RETURN_ON_FAIL(addEmbeddedLibrary(CodeGenTarget::SPIRV));
        break;
```